### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by mfranco**
@niallryan @Rondineli @cormacp @vcali

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,78 @@
...
+[here](http://flask-philo.readthedocs.io/en/latest/)
 
-## What is this project?
 
-Flask is an awesome web microframework that works great out of the box. Nevertheless,
-additional configuration and integration with complementary libraries are required
-in order to build complex applications. Here in [riffstation.com](https://play.riffstation.com) we have several
-HTTP REST based microservices, anytime that we want to create a new microservice we need to bootstrap
-very similar common code.
+## What this project is about?
```
Comment (cormacp) [inline README.md]: Recommend correction : "What is this project", or "What is this project about"

File: README.md [hunk 2]
```diff
@@ -1,21 +1,78 @@
 # Flask-Philo
 
-Flask-Philo is a small and very opinionated framework to build Flask-based microservices.
+![Flask-Philo Logo](https://raw.githubusercontent.com/Riffstation/flask-philo/dev/documentation/source/_static/img/banner_1.png)
 
-You can check the official documentation [here](http://flask-philo.readthedocs.io/en/latest/)
+Small and very opinionated framework to build flask based microservices.
```
Comment (cormacp) [inline README.md]: Recommended correction : "A small and very opinionated..."

File: README.md [hunk 3]
```diff
@@ -1,21 +1,78 @@
...
+
+Having the container id you can run the unit tests using the following command:
+
+
+```
+python3 run_tests.py --container_id container-id   test --test tests/
+```
 
 
 ## Basic Features
```
Comment (cormacp) [inline README.md]: Would it make more sense to include "Basic Features" at the end of the Introduction paragraph?

File: README.md [hunk 4]
```diff
@@ -1,21 +1,78 @@
...
+## Installation
+
+Flask-Philo installation is pretty straightforward:
+
+```
+pip3 install Flask-Philo
+```
+
+
+## Executing test
```
Comment (cormacp) [inline README.md]: Recommended correction : "Executing Tests" or "Executing Unit Tests"

File: README.md [hunk 5]
```diff
@@ -1,21 +1,78 @@
...
+
+The following command creates the docker images required for running the tests:
+
+```
+cd test
+docker-compose up --build
+```
+
+If you get an error related with ``__pycache__``  directories or ``pyc`` files,
+you need to remove  python bytecode generated by running the following command:
```
Comment (cormacp) [inline README.md]: Recommended correction : "...you need to remove generated Python bytecode by running..."

File: README.md [hunk 6]
```diff
@@ -1,21 +1,78 @@
...
+
+```
+find . | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs sudo rm -rf
+```
+
+
+After building and running the docker images you will need to get the
+container id for the maching labeled as ``test_python`` by running the following
```
Comment (cormacp) [inline README.md]: Correction "maching labeled as" -> "machine labelled"

File: README.md [hunk 7]
```diff
@@ -1,21 +1,78 @@
...
+command:
+
+```
+docker ps
+
```
Comment (cormacp) [inline README.md]: remove linebreak

File: README.md [hunk 8]
```diff
@@ -34,13 +89,20 @@ Feel free to use it and extend it. We are willing to hear about your suggestions
 
 
 
-## Installation
+## External Resources
```
Comment (cormacp) [inline README.md]: It may be better to place the "External Resources" section at the end of the Readme

Comment (cormacp) [reply README.md]: ...Actually, I think the readme already includes an "External Resources" section, so this is duplication

**Comment by mfranco** [comment]
:+1:

**Pull request merged**